### DISCUSSION TITLE
Hybrid AX Pro

### DISCUSSION THREAD

**Discussion by timurvafin** [petrleocompel/hikaxpro_hacs#77] [Q&A] [answered]
Do you support Hybrid panels like AX PRO Hybrid DS-PHA48-EP?

**Comment by petrleocompel** [comment, marked as answer]
Yes.

**Reply by timurvafin** [reply]
Sound good, going to install it with wired sensors and test addon

**Reply by petrleocompel** [reply]
I am using the higher version of this. With wired sensors.

**Reply by timurvafin** [reply]
Btw, don you know what is the difference between: DS-PHA and DS-PWA?

**Reply by petrleocompel** [reply]
I think it is region difference.
